Project.toml

```diff
@@ -7,6 +7,7 @@ version = "1.0.3"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
 DistributedData = "f6a0035f-c5ac-4ad0-b410-ad102ced35df"
+DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
```

Comment:
was the formatting thing actually fixed?

Reply:
(if not, let's not add DSEs yet)

Reply:
Formatting not fixed, but the formatting only looks ugly for long docstrings, for shorter ones it looks good.

Reply:
but you decide, in or out?

Reply:
I would keep it in, since it is kinda nice